src/docs/Glossary.adoc

```diff
@@ -90,22 +94,74 @@ of a <<_glossary_workflow,*Workflow*>>.
+
+[[_glossary_task_executable]]
+Task Executable::
+  The main executable definition of a <<_glossary_task,*ProActive Task*>>. A Task Executable can either be a <<_glossary_task_executable_script,*Script Task*>>, a <<_glossary_task_executable_java,*Java Task*>> or a <<_glossary_task_executable_native,*Native Task*>>.
+
+[[_glossary_task_executable_script]]
+Script Task::
+  A <<_glossary_task_executable,*Task Executable*>> defined as a script execution.
+
+[[_glossary_task_executable_java]]
+Java Task::
+  A <<_glossary_task_executable,*Task Executable*>> defined as a Java class execution.
+
+[[_glossary_task_executable_native]]
+Native Task::
+  A <<_glossary_task_executable,*Task Executable*>> defined as a native command execution.
+
+[[_glossary_additional_task_scripts]]
+Additional Task Scripts::
+   A collection of scripts part of a <<_glossary_task,*ProActive Task*>> definition which can be used in complement to the main <<_glossary_task_executable,*Task Executable*>>. Additional Task scripts can either be <<_glossary_selection_script,*Selection Script*>>, <<_glossary_fork_environment_script,*Fork Environment Script*>>, <<_glossary_pre_script,*Pre Script*>>, <<_glossary_post_script,*Post Script*>>, <<_glossary_control_flow_script,*Control Flow Script*>> or <<_glossary_clean_script,*Cleaning Script*>>
+
+[[_glossary_selection_script]]
+Selection Script::
+  A script part of a <<_glossary_task,*ProActive Task*>> definition and used to select a specific <<_glossary_proactive_node,*ProActive Node*>> to execute a <<_glossary_task,*ProActive Task*>>.
+
+[[_glossary_fork_environment_script]]
+Fork-Environment Script::
+  A script part of a <<_glossary_task,*ProActive Task*>> definition and run on the <<_glossary_proactive_node,*ProActive Node*>> selected to execute the Task to configure the forked Java Virtual Machine to execute the task.
+
+[[_glossary_pre_script]]
+Pre Script::
+  A script part of a <<_glossary_task,*ProActive Task*>> definition and run inside the forked Java Virtual Machine, before the <<_glossary_task_executable,*Task Executable*>>.
+
+[[_glossary_post_script]]
+Post Script::
+  A script part of a <<_glossary_task,*ProActive Task*>> definition and run inside the forked Java Virtual Machine, after the <<_glossary_task_executable,*Task Executable*>>
+
+[[_glossary_control_flow_script]]
+Control Flow Script::
+  A script part of a <<_glossary_task,*ProActive Task*>> definition and run inside the forked Java Virtual Machine, after the <<_glossary_task_executable,*Task Executable*>>, to determine control flow actions.
+
+[[_glossary_clean_script]]
+Cleaning Script::
+  A script part of a <<_glossary_task,*ProActive Task*>> definition and run after the <<_glossary_task_executable,*Task Executable*>> and before releasing the <<_glossary_proactive_node,*ProActive Node*>> to the <<_glossary_resource_manager,*Resource Manager*>>.
```

Comment:
We do not say here in which JVM it is executed (as opposed to the previous script definitions). Maybe we could generalize a standard explanation format for all script definitions (you do it quite a lot already), e.g. where it is written, when it is run, in which JVM, on which ProActive Node.

Reply:
this is the scope of a dedicated documentation section. Here, I simply update the glossary. I am aware that the current documentation regarding task execution is not enough and many restrucurations must be perfomed, but this will be for later